packages/wdio-config/src/node/ConfigParser.ts

```diff
@@ -478,4 +478,10 @@ export default class ConfigParser {
         const end = current === total ? undefined : specsPerShard * current
         return specs.slice(current * specsPerShard - specsPerShard, end)
     }
-}
+
+    cliSpecArgContainsPathstoSpecs(specArr: string[]) {
+    // check that CLI --spec arg does not contain path to file
+    // If arg contains '/', '\' or '.' => in this case it is filepath
+        return /[/\\.]/.test(specArr[0])
```

Comment:
I think we can simplify this, @christian-bromann what do you think about this one?
```suggestion
        return specArr[0].includes(path.sep)
```

Reply:
Yes, and I think there is not need for making this a function since we don't use this anywhere else

Reply:
Method has been updated